Source/Editor/Foundation/InspectorTab/ModelInspector.cpp

```diff
@@ -50,14 +51,70 @@ SharedPtr<ResourceInspectorWidget> ModelInspector::MakeInspectorWidget(const Res
     return MakeShared<ModelInspectorWidget>(context_, resources);
 }
 
+class ModelInspectorSceneWidget : public SceneWidget
+{
+public:
+    ModelInspectorSceneWidget(Context* context)
+        : SceneWidget(context)
+    {
+    }
+
+    ~ModelInspectorSceneWidget() {}
+
+    void SetModel(Model* model)
+    {
+        model_ = model;
+    }
+
+    void RenderContent() override
+    {
+        if (model_)
+        {
+            auto& vertexBuffers = model_->GetVertexBuffers();
+            if (!vertexBuffers.empty())
+            {
+                ui::NewLine();
```

Comment:
I would suggest not inserting a new line to save space.

Reply:
Done